core/modules/user/tests/user.test

```diff
@@ -1044,7 +1044,7 @@ class UserPictureTestCase extends BackdropWebTestCase {
     // Try to upload a file that is not an image for the user picture.
     $not_an_image = current($this->backdropGetTestFiles('html'));
     $this->saveUserPicture($not_an_image);
-    $this->assertRaw(t('Only JPEG, PNG and GIF images are allowed.'), 'Non-image files are not accepted.');
+    $this->assertRaw(t('The image type is not recognized.'), 'Non-image files are not accepted.');
```

Comment:
This doesn't seem to solve the problem (all allowed extensions are not mentioned)

Reply:
The task I created is to remove the image types in that error message, since `file_validate_image()` returns TRUE for all the images recognized from the enabled image toolkit. For GD with enabled support for WebP images, that includes that image too. The issue title has not been changed by me; the original title was different.

Reply:
In the issue has been then suggested that the error message should reflect what value is returned by `image_get_info()`: `FALSE` for file not found, an array with `'extension'` set to `''` for not recognized images. (Truly, `FALSE` is also returned when the file has not been uploaded via HTTP POST.)  
We are still discussing about the exact error message.